config/configLoader.go

```diff
@@ -20,6 +20,10 @@ import (
 	"github.com/spf13/viper"
 )
 
+var (
+	SupportedFormats = []string{"toml", "yaml", "yml", "json"}
```

Comment:
The change itself is good, but I think it would be more correct if this variable said, perhaps,`ValidConfigFileExtensions` -- esp. with the duplicate YAML definition.